integration/utils_test.go

```diff
@@ -329,19 +332,17 @@ func fakeTar() (io.ReadCloser, error) {
 	return ioutil.NopCloser(buf), nil
 }
 
-func getImages(eng *engine.Engine, t *testing.T, all bool, filter string) *engine.Table {
-	job := eng.Job("images")
-	job.SetenvBool("all", all)
-	job.Setenv("filter", filter)
-	images, err := job.Stdout.AddListTable()
-	if err != nil {
-		t.Fatal(err)
+func getImages(eng *engine.Engine, t *testing.T, all bool, filter string) []types.Image {
+	config := graph.ImagesConfig{
+		Filter: filter,
+		All:    all,
 	}
-	if err := job.Run(); err != nil {
+	images, err := server.GetDaemon(eng).Repositories().CmdImages(&config)
```

Comment:
Can we just copy behavior of `getDaemon` here? Because I'm not happy with exposing `getDaemon` from server just for integration tests. Also this should reduce size of patch.